README.md

```diff
@@ -202,6 +202,38 @@ bind -r C-l run "tmux select-pane -R"
 bind -r C-\ run "tmux select-pane -l"
 ```
 
+#### Disable Wrapping
+
+By default, if you try to move past the edge of the screen, tmux/vim will
+"wrap" around to the opposite side. To disable this, you'll need to
+configure both tmux and vim:
+
+
+For vim, you only need to enable this option:
+``` vim
+let g:tmux_navigator_no_wrap = 1
+```
+
+Tmux doesn't have an option, so whatever key bindings you have need to be set
+to conditionally wrap based on position on screen:
+``` tmux
+# Smart pane switching with awareness of Vim splits.
```

Comment:
This really should also go into [vim-tmux-navigator.tmux](https://github.com/christoomey/vim-tmux-navigator/blob/master/vim-tmux-navigator.tmux)